docs/modules/postgres.md

```diff
@@ -51,7 +51,7 @@ If you need to set a different database, and its credentials, you can use the `W
 
 #### Init Scripts
 
-If you would like to do additional initialization in the Postgres container, add one or more `*.sql`, `*.sql.gz`, or `*.sh` scripts to the container request with the `WithInitScript` function.
+If you would like to do additional initialization in the Postgres container, add one or more `*.sql`, `*.sql.gz`, or `*.sh` scripts to the container request with the `WithInitScripts` function.
```

Comment:
When I saw typo in postgres readme, I wanted to fix this typo in this pr also.

Reply:
Please do it in another PR as it is outside the current scope.